proposals/0111-linear-types.rst

```diff
@@ -297,40 +297,21 @@ This proposal adds two new syntactical constructs:
 
     \x %'One -> x
 
-  This modifies the syntax entry for pattern with signature annotation
-  as follows as follows
+  This modifies the syntax entry for lambda expressions
+  as follows
 
   ::
 
-    pat -> pat [PREFIX_PERCENT btype] [:: type]
+    lexp -> \ apat1 ... apatn [PREFIX_PERCENT btype] -> exp
 
   where the ``btype`` after the ``%`` must be of kind ``Multiplicity``
   (see below).
 
-  This form is disallowed for:
-
-  - Type variables
-
-    ::
-
-      forall (a %'One). a -> Int -- rejected
-  - Top-level signatures (though, see `Toplevel binders`_)
-
-    ::
-
-      foo %'One :: A -> B -- rejected
-      foo x = …
-
-  The form is however permitted in records (see `Records`_ below)
-
-  ::
-
-    data R = R { unrestrictedField %'Many :: A, linearField %'One :: B }
-
-  This modifies the field declaration syntax to
-
-  ::
+  This form is allowed only when the lambda binds precisely one variable.
```

Comment:
Why? This sounds arbitrary.

Reply:
I think I didn't want the multiplicity to apply to all bound variables / patterns, but it seems, in retrospect, unnecessary.